src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -230,6 +232,11 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 3,
                 "No actions matched the current request. Route values: {RouteValues}");
+
+            _redirectToPageResultExecuting = LoggerMessage.Define<string>(
+                LogLevel.Information,
+                5,
```

Comment:
Should this be `1`? Seems like that's the eventId for all the other redirect methods declared here.